packages/file_selector/file_selector_web/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.9.4+2
+
+* Supports `web: ">=0.5.1 <2.0.0"`.
+* Updates SDK version to Flutter `^3.19.0`.
```

Comment:
The Flutter version doesn't appear to have changed. I think we can remove this line.

Reply:
Yes! This is a typo from `0.9.4`: https://github.com/flutter/packages/commit/b21dce5f4964adf1e6c90225377a00f65652c428#diff-a1970a9bbbd9cd744445641d16604a32f71cb82413a2dc02d216eacd5101616d (that I'm about to retcon)